sklearn/ensemble/tests/test_bagging.py

```diff
@@ -755,6 +755,12 @@ def test_set_oob_score_label_encoding():
     assert_equal([x1, x2], [x3, x3])
 
 
+def replace(X):
+    X = X.copy().astype('float')
```

Comment:
Sorry to necro this, but `astype` copies by default. Should there still be a `copy` call here? Admittedly I know nothing about this code. So feel free to correct me if I'm missing something.

Reply:
you're right. This is a hack used in a test, though, so I don't think it matters. We could remove the ``copy()`` though.